### PR TITLE
Adding test in Travis.

### PR DESCRIPTION
**Pull request by raquel-ucl**

### CODE REVIEW

**Comment by jenshnielsen** [comment]
I don't think the OSX env is the solution thats mainly for OSX specific packages. I would suggest trying the trusty env here https://docs.travis-ci.com/user/ci-environment/

**Comment by raquel-ucl** [comment]
It the mvn install worked fine with trusty, but the mvn test keeps on breaking.

**Comment by raquel-ucl** [comment]
I've run it again and now the mvn install breaks, it is still very unstable. I'm not sure what else to try. Should we stick to jenkins-only tests by now?

**Comment by jenshnielsen** [comment]
Yes leave this for the time being

**Comment by jenshnielsen** [comment]
You are still not running on the Trusty env it has to be 

```
sudo: required
dist: trusty
```
for that to work. The log file with tell you that you are running on the container build

**Comment by jenshnielsen** [comment]
And the tests are backlogged as usual on the GCE instances.

**Comment by jenshnielsen** [comment]
retest this please

**Comment by jenshnielsen** [comment]
retest this please

**Comment by jenshnielsen** [comment]
retest this please

**Comment by raquel-ucl** [comment]
Ready to merge, I think.

File: pom.xml [hunk 1]
```diff
@@ -25,7 +25,7 @@
 		<dependency>
 			<groupId>org.python</groupId>
 			<artifactId>jython-standalone</artifactId>
-			<version>2.7.0</version>
+			<version>2.7.1b3</version>
```
Comment (raquel-ucl) [inline pom.xml]: This might not be needed.

Comment (jenshnielsen) [inline pom.xml]: Lets leave it. Then it matches the version in the plugin.

**Comment by jenshnielsen** [comment]
Replaced by #96 (added more tests and cleaned up a bit)

**Pull request closed without merging**